src/helpers/mysql.ts

```diff
@@ -1,11 +1,20 @@
 import mysql from 'mysql';
+// @ts-ignore
 import Pool from 'mysql/lib/Pool';
+// @ts-ignore
 import Connection from 'mysql/lib/Connection';
```

Comment:
maybe we should update mysql version to avoid this? if we update the connection code like here https://github.com/snapshot-labs/keycard/blob/master/src/helpers/mysql.ts

Reply:
@wa0x6e Also check this :)

Reply:
This is not fixable, as those file are not typed, since not designed to be exposed and used like this.

But instead, we can mute those error by another method, and avoid having `// @ts-ignore` in the code